connector/src/main/scala/com/microsoft/kusto/spark/utils/KustoClient.scala

```diff
@@ -118,67 +124,87 @@ class KustoClient(val clusterAlias: String, val engineKcsb: ConnectionStringBuil
     extentsLeftRes.getInt(0) != 0
   }
 
+  def findErrorInResult(res: KustoResultSetTable): (Boolean, Object) = {
+    var error:Object = null
+    var failed = false
+    if (KDSU.getLoggingLevel == Level.DEBUG) {
+      var i = 0
+      while (res.next() && !failed) {
+        val targetExtent = res.getString(1)
+        error = res.getObject(2)
+        if (targetExtent == "Failed" || StringUtils.isNotBlank(error.asInstanceOf[String])) {
+          failed = true
+          if (i > 0) {
+            KDSU.logFatal(myName, "Failed extent was not reported on all extents!." +
```

Comment:
This is enclosed in the Debug clause, yet seems like you expect customers to see and respond to this alert

Reply:
My intention here was that as this was fast written and tested that if user complains he would turn the logging to debug and send us complete logs after that